Octokit.Tests.Integration/Clients/RepositoryInvitationsClientTests.cs

```diff
@@ -70,6 +179,158 @@ public class TheGetAllForCurrentMethod
                 Assert.NotNull(invitations.FirstOrDefault(i => i.Repository.Id == response.Repository.Id));
             }
         }
+
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfInvitationsWithStart()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoNames = Enumerable.Range(0, 6).Select(i => Helper.MakeNameWithTimestamp($"public-repo{i}")).ToList();
+
+            var contexts = new List<RepositoryContext>();
+            try
+            {
+                foreach (var repoName in repoNames)
+                {
+                    contexts.Add(await github.CreateRepositoryContext(new NewRepository(repoName)));
+                }
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest(Permission.Push);
+
+                // invite a collaborator to all repos
+                foreach (var context in contexts)
+                {
+                    var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, context.RepositoryOwner, permission);
+
+                    Assert.Equal(context.RepositoryOwner, response.Invitee.Login);
+                    Assert.Equal(InvitationPermissionType.Write, response.Permissions);
+                }
+
+                var startOptions = new ApiOptions
+                {
+                    PageSize = 5,
+                    PageCount = 1,
+                    StartPage = 2
+                };
+
+
+                var invitations = await github.Repository.Invitation.GetAllForCurrent(startOptions);
+                Assert.Equal(1, invitations.Count);
+            }
+            finally
+            {
+                if (contexts != null)
+                {
+                    foreach (var context in contexts)
+                    {
+                        context?.Dispose();
+                    }
+                }
+            }
+        }
+
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfInvitationsWithoutStart()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoNames = Enumerable.Range(0, 6).Select(i => Helper.MakeNameWithTimestamp($"public-repo{i}")).ToList();
```

Comment:
Let's use pagesize=1 and just create 2 test repos

Reply:
Done.